app/src/main/kotlin/com/looker/droidify/ui/appList/AppListViewModel.kt

```diff
@@ -37,6 +37,8 @@ class AppListViewModel
     val sortOrderFlow = settingsRepository.get { sortOrder }
         .asStateFlow(SortOrder.UPDATED)
 
+    val settingsRepository = settingsRepository
```

Comment:
I dont think this will be needed after the changes I suggested, but normally you can just add `val` to the constructor parameter and it will be accessible just like this 😄